services/dedup/dedup.go

```diff
@@ -30,33 +23,8 @@ func DefaultPath() string {
 
 // New creates a new deduplication service. The service needs to be closed after use.
 func New(path string) Dedup {
```

Comment:
[optional] I guess we can remove path from the input as it might be more relevant to badger and might not be relevant to some other DB.

Reply:
I wanted to remove this but I forgot. Will remove this